catalogue_graph/src/ingestor/transformers/raw_concept.py

```diff
@@ -170,9 +172,15 @@ def identifiers(self) -> list[ConceptIdentifier]:
     @property
     def display_identifiers(self) -> list[DisplayIdentifier]:
         display_ids = []
+
         for identifier in self.identifiers:
+            source_identifier = SourceIdentifier(
+                value=identifier.value,
+                identifier_type=Id(id=identifier.identifierType),
+                ontology_type="Concept",
+            )
             display_ids.append(
-                get_display_identifier(identifier.value, identifier.identifierType)
+                DisplayIdentifier.from_source_identifier(source_identifier)
             )
 
         return display_ids
```

Comment:
I think this could be a comprehension:
```suggestion
return [
        DisplayIdentifier.from_source_identifier(
            source_identifier = SourceIdentifier(
                value=identifier.value,
                identifier_type=Id(id=identifier.identifierType),
                ontology_type="Concept",
            )
        ) for identifier in self.identifiers
    ]
```